Practicum/Pract. 08/StringVector.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include <iostream>
+#include "String.h"
+
+#define DEFAULT_CAPACITY 8
```

Comment:
better have this inside the cpp if it won't be needed anywhere else.

Also, after moving it to the cpp it will be better to write it in any of the following ways:

- static variable
- static variable in the class
- namespace
- anonymous namespace